middleware/kubernetes/handler_test.go

```diff
@@ -81,6 +81,22 @@ var dnsTestCases = map[string](*test.Case){
 			test.CNAME("svc0.testns.fed.svc.cluster.local.	0	IN	CNAME	svc0.testns.fed.svc.fd-az.fd-r.federal.test."),
 		},
 	},
+	"AAAA Service (existing service)": {
```

Comment:
Are we doing the correct thing for other types as well? i.e. MX lookups or other things that might get queried?